### PR TITLE
3214 - Files and links per Cycle: remove unused and deprecated code

### PR DESCRIPTION
**Pull request by sorja**
Added new todos in main PR:

```
 Get hidden files ( getHiddenFile.ts )
Update import private files tools/importPrivateFiles/index.ts
```

### CODE REVIEW

File: src/server/api/file/getHiddenFile.ts [hunk 1]
```diff
@@ -2,21 +2,21 @@ import { Request, Response } from 'express'
...
   try {
-    const { assessmentName, fileName } = req.query
+    // const { assessmentName, fileName } = req.query
 
-    const assessment = await AssessmentController.getOne({ assessmentName })
-    const assessmentFile = await FileController.getHiddenAssessmentFile({ assessment, fileName })
-
-    Responses.sendAssessmentFile(res, assessmentFile)
+    // const assessment = await AssessmentController.getOne({ assessmentName })
+    // TODO
```
Comment (minotogna) [inline src/server/api/file/getHiddenFile.ts:16]: has the url of hidden file changed? with this I think we shouldn't change it, or at least ask where they are user

File: src/tools/importPrivateFiles/index.ts [hunk 2]
```diff
@@ -7,9 +7,11 @@ import * as path from 'path'
 import { AssessmentNames } from 'meta/assessment'
 
 import { AssessmentController } from 'server/controller/assessment'
-import { AssessmentFileRepository } from 'server/repository/assessment/file'
 import { Logger } from 'server/utils/logger'
 
+// TODO
```
Comment (minotogna) [inline src/tools/importPrivateFiles/index.ts]: what is this ? 😅

Comment (sorja) [reply src/tools/importPrivateFiles/index.ts]: Should we remove this tool?
I added it for now, let's implement when it's needed again?

File: src/server/api/file/getHiddenFile.ts (hunk 1 above)
Comment (sorja) [reply src/server/api/file/getHiddenFile.ts:16]: Should we introduce redirect with `HTTP 301`?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301

File: src/tools/importPrivateFiles/index.ts (hunk 2 above)
Comment (minotogna) [reply src/tools/importPrivateFiles/index.ts]: oh right, totally forgot about this ;P . sure let's get rid of it for now. we might even build an admin page where they can do this from there directly. what do u think ?

File: src/server/api/file/getHiddenFile.ts (hunk 1 above)
Comment (minotogna) [reply src/server/api/file/getHiddenFile.ts:16]: yes, I think so. those links where shared with people. another pr ?

Comment (sorja) [reply src/server/api/file/getHiddenFile.ts:16]: Yes

**Review by minotogna — APPROVED** [review]

**Pull request merged**